lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -368,13 +367,17 @@ void WebAssemblyInstance::LoadImports(
             {
                 if (!WebAssemblyTable::Is(prop))
                 {
-                    JavascriptError::ThrowWebAssemblyLinkError(ctx, WASMERR_NeedTableObject);
+                    throw Wasm::WasmCompilationException(_u("Invalid import '%s.%s'. Expected a WebAssembly.Table"), import->modName, import->importName);
```

Comment:
How is the localization work for these strings?

---
In reply to: [131437934](https://github.com/Microsoft/ChakraCore/pull/3487#discussion_r131437934) [](ancestors = 131437934)